### PR TITLE
add a link to "meeting contact person"

### PR DESCRIPTION
**Pull request by yymao**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -124,7 +124,8 @@ <h1>LSST DESC registration form </h1>
       <div class="form-check">
         <input class="form-check-input" type="checkbox" id="contact" name="contact">
         <label class="form-check-label" for="contact">
-        Check if you do NOT wish to be considered for the role of meeting contact person.
+        Check if you do NOT wish to be considered for the role of 
+          <a href="http://lsstdesc.org/sites/default/files/LSST_DESC_Professional_Conduct.pdf#page=11" target="_top">Meeting Contact Persons</a>.
```
Comment (EiffL) [inline index.html]: Thanks for the change, actually, Rachel pointed out that it's not just a question of willingness but also of restrictions on who can be a contact person:  “… or are not eligible to serve in this role”

Comment (yymao) [reply index.html]: So the worry is that people may not know whether or not they are eligible.  The MeetComm will remove those who are not eligible via an automated script. (cc @rmandelb)

**Comment by yymao** [comment]
@EiffL i don't think most people know what the requirements are... isn't this going to be confusing?

**Comment by EiffL** [comment]
hum, yes I guess I'm fine with not mentionning it and have it sorted out programatically afterwards :-) 
Otherwise, if there is a short description of the requirements, I can display it as a floating text that appears when the mouse passes over the text

**Comment by EiffL** [comment]
Ok, sorry I tried to roll back the modifications and I made a mess, cant reopen your PR

**Comment by yymao** [comment]
Well, a person is eligible if they are a DESC Full Member but are not part of the Council and are not in the Management Team (= Spokespersons, Coordinators, Operation Manager and the Operation Committee)...

**Pull request merged**